triggers/scheduler.py

```diff
@@ -1,17 +1,18 @@
+import asyncio
 import logging
+from asyncio import run as aiorun
 
 import typer
+from apscheduler.events import EVENT_JOB_ERROR, EVENT_JOB_EXECUTED
```

Comment:
Had to replace `rocketry `with `apscheduler` as `rocketry `has a dependency on `pydantic`. Currently `rocketry `is using `pydantic `v1.